Automattic/jetpack
# Fix/swiper css specificity issues

Fixes #20579

#### Changes proposed in this Pull Request:

* Adds a jetpack specific selector to the swiperjs css to prevent clashes with other plugins that use swiper.

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

* Check out this PR to local dev and check that all the css still operates as expected on desktop, mobile and RTL
